builtin/providers/azurerm/resource_arm_virtual_machine_test.go

```diff
@@ -475,6 +475,13 @@ resource "azurerm_virtual_machine" "test" {
     	environment = "Production"
     	cost-center = "Ops"
     }
+
+    diagnostics_profile {
```

Comment:
Please can we remove the diagnostics_profile from all tests and add a specific test for that functionality. This will help us to make sure that our tests are targetted towards specific test types :)